apis/v1alpha2/httproute_types.go

```diff
@@ -349,6 +349,9 @@ type HTTPHeaderMatch struct {
 	// case-insensitivity of header names, "foo" and "Foo" are considered
 	// equivalent.
 	//
+	// Matching behavior for headers with multiple values (like `Forwarded-for`)
+	// is not defined by this spec.
+	//
```

Comment:
> Subsequent entries with an equivalent header name MUST be ignored.

So.. is someone going to change envoy to match like this or are we all just going to be non-compliant :slightly_smiling_face: